app/components/stepper-signup.js

```diff
@@ -16,15 +20,23 @@ export default class StepperSignupComponent extends Component {
     );
   }
 
+  @action decrementStep() {
+    if (this.currentStep > 0) {
+      this.currentStep -= 1;
+      const queryParams = { dev: true, step: this.currentStep };
+      this.router.transitionTo('join', { queryParams });
+    }
+  }
+
   @action incrementStep() {
-    if (this.currentStep < 5) {
+    if (this.currentStep < 10) {
```

Comment:
Can we have this 10 in constant?
MAX_STEP = 10

Reply:
done